designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -115,16 +116,51 @@ object ScenarioStatistics {
     }
   }
 
-  def getComponentStatistic(componentList: List[component.ComponentListElement]): Map[String, String] = {
+  def getComponentStatistic(
+      componentList: List[component.ComponentListElement],
```

Comment:
Can we leave todo comment about removing components from restmodel?

Reply:
Why do we still use DTO's (`ComponentListElement`) in the domain logic, we should base on only domain objects (`ComponentDefinitionWithImplementation`)

Reply:
We use `ComponentListElement` here to get `usageCount` for each component. I knew that it can be provided by `ComponentService` so I did it this way, but if there's another way I can change it. Or if there is currently no other way, I can try to make one